.github/scripts/run_consensus.sh

```diff
@@ -83,7 +83,7 @@ function generate_chainspec() {
 
   echo "Generate chainspec and keystores for accounts: ${account_ids_comma_separated[@]}"
   docker run --rm -v $(pwd)/docker/data:/data --entrypoint "/bin/sh" -e RUST_LOG=debug "${CHAIN_BOOTSTRAPPER_IMAGE}" \
-  -c "chain-bootstrapper bootstrap-chain --base-path /data --account-ids ${account_ids_comma_separated} --authorities-account-ids ${validator_ids_comma_separated}  > /data/chainspec.json"
+  -c "chain-bootstrapper bootstrap-chain --base-path /data --account-ids ${account_ids_comma_separated} --authorities-account-ids ${validator_ids_comma_separated} --raw > /data/chainspec.json"
```

Comment:
Not related, although in the process of working on this PR I thought this was. I left it added since in `run_nodes.sh` we also haw raw chain spec, so to unify approaches across dev env.